packages/syft/src/syft/ast/klass.py

```diff
@@ -138,7 +146,10 @@ def run_class_method(
 
             # then we convert anything which isnt a pointer into a pointer
             pointer_args, pointer_kwargs = pointerize_args_and_kwargs(
-                args=downcast_args, kwargs=downcast_kwargs, client=__self.client
+                args=downcast_args,
+                kwargs=downcast_kwargs,
+                client=__self.client,
+                gc_enabled=False,
```

Comment:
Q: Why do we disable garbage collection?

Reply:
The same reason as bellow